rexray/cli/cli.go

```diff
@@ -347,6 +347,10 @@ func (c *CLI) preRun(cmd *cobra.Command, args []string) {
 		if c.runAsync {
 			c.ctx = c.ctx.WithValue("async", true)
 		}
+		if !c.config.IsSet("docker.mountDirPath") {
+			c.config.Set("docker.mountDirPath", "/var/lib/rexray/volumes")
```

Comment:
Hi @clintonskitson,

The value should be `util.LibFilePath("volumes")` where `util` is the package `github.com/emccode/rexray/util`